src/website/app/demos/Dropdown/examples/scrolling.js

```diff
@@ -23,7 +23,7 @@ import data from '../../Menu/components/menuData';
 export default {
   id: 'scrolling-container',
   title: 'Scrolling container',
-  description: 'Behavior inside of a scrolling container.',
+  description: `Dropdown will react to an \`overflow: scroll\` container in the same way as to the viewport by re-orienting in relation to the trigger.`,
```

Comment:
Unfortunately, this is false. Try:

"Dropdown will attempt to keep its menu visible when placed inside a scroll container."?

Reply:
Otherwise, maybe just the simple description it had before?

Reply:
ah. updated in the Popover scrolling example too.